_includes/cloudcode/cloud-code.md

```diff
@@ -319,3 +319,57 @@ The `afterDelete` handler can access the object that was deleted through `reques
 The client will receive a successful response to the delete request after the handler terminates, regardless of how it terminates. For instance, the client will receive a successful response even if the handler throws an exception. Any errors that occurred while running the handler can be found in the Cloud Code log.
 
 If you want to use `afterDelete` for a predefined class in the Parse JavaScript SDK (e.g. [Parse.User](https://parse.com/docs/js/api/symbols/Parse.User.html)), you should not pass a String for the first argument. Instead, you should pass the class itself.
+
+# beforeFind Triggers
+
+*Available only on parse-server cloud code*
```

Comment:
Please mention which version of Parse Server is the minimum required for this to work.